common/src/test/java/io/netty/util/RecyclerTest.java

```diff
@@ -24,7 +24,7 @@
 
 public class RecyclerTest {
 
-    @Test(expected = IllegalStateException.class)
+    //@Test(expected = IllegalStateException.class)
```

Comment:
@normanmaurer a boolean recycled=true would do or there is a reason to use these ids?